importer/helpers/dagbuilder.go

```diff
@@ -139,7 +155,11 @@ func (db *DagBuilderHelper) GetNextDataNode() (*UnixfsNode, error) {
 			raw:     true,
 		}, nil
 	} else {
-		blk := NewUnixfsBlock()
+		blk := &UnixfsNode{
```

Comment:
If you want I can turn it into a method like I did for `NewUnixfsNode`.  But since it was only used in one place I decided it wasn't worth the effort.